sohyun/EDA_Clean_sohyun.ipynb

```
@@ -0,0 +1,1109 @@
+    "!apt-get install -y fonts-nanum"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 154,
+   "id": "e88aec49",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# visualization\n",
+    "import matplotlib.pyplot as plt\n",
```

Ruff (0.12.2)

`sklearn.preprocessing.OneHotEncoder` imported but unused

Remove unused import: `sklearn.preprocessing.OneHotEncoder`

(F401)